distributed/stealing.py

```diff
@@ -178,6 +179,14 @@ def move_task_request(self, key, victim, thief):
                                    'thief': thief,
                                    'victim_duration': victim_duration,
                                    'thief_duration': thief_duration}
+            try:
+                self.in_flight_occupancy[victim] -= victim_duration
+            except KeyError:
+                self.in_flight_occupancy[victim] = -victim_duration
+            try:
+                self.in_flight_occupancy[thief] += thief_duration
+            except KeyError:
+                self.in_flight_occupancy[thief] = thief_duration
```

Comment:
It might make sense to define `self.in_flight_occupancy = defaultdict(lambda: 0)` instead of using these try-excepts.

Reply:
Agreed.